recipe/meta.yaml

```diff
@@ -13,10 +13,12 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
-  number: 0
+  number: 1
   script: 
     - python setup.py install --record=record.txt  # [not py3k]
     - python setup3.py install --record=record.txt  # [py3k]
+    - copy %PREFIX%\Lib\site-packages\pywin32_system32\*.dll %PREFIX%
```

Comment:
No.  DLLs always go into %LIBRARY_BIN%.  %LIBRARY_LIB% is for .lib files, which on Windows are either import libraries or static libraries.

%LIBRARY_BIN% is on PATH, %LIBRARY_LIB% is not.